### ISSUE TITLE
coco_eval error AP=0

### ISSUE THREAD

**Issue by dailybudushu** [zylo117/Yet-Another-EfficientDet-Pytorch#136]
Everything works fine when I trained the model.
I got the following result when running coco_val:
```
running coco-style evaluation on project coco_helmet, weights logs/coco_helmet-C4/efficientdet-d4_9_31562.pth...
loading annotations into memory...
Done (t=0.12s)
creating index...
index created!
Loading and preparing results...
DONE (t=1.29s)
creating index...
index created!
BBox
Running per image evaluation...
Evaluate annotation type *bbox*
DONE (t=34.98s).
Accumulating evaluation results...
DONE (t=1.42s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.001
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.002

```
 loss:
![image](https://user-images.githubusercontent.com/45378108/79642017-a3867c00-81cd-11ea-8fc0-c1ca868ea386.png)

 
![image](https://user-images.githubusercontent.com/45378108/79642039-c749c200-81cd-11ea-9f55-af08c7f6fe49.png)

![image](https://user-images.githubusercontent.com/45378108/79642056-db8dbf00-81cd-11ea-81bb-ce05cd92be3f.png)
![image](https://user-images.githubusercontent.com/45378108/79642065-ea747180-81cd-11ea-92d9-7cb987d65ae5.png)

**Comment by zylo117** [comment]
1. latest code?
2. succeeded running the tutorial and get a decent result?
3. Can't see any val loss on the graph. Is it because your weights is overfitting so that val loss is too high to see?

**Comment by dailybudushu** [comment]
> 1. latest code?
> 2. succeeded running the tutorial and get a decent result?
> 3. Can't see any val loss on the graph. Is it because your weights is overfitting so that val loss is too high to see?

thanks, there are something wrong with my datasets.
I have already fixed that

**Comment by qtw1998** [comment]
> > 1. latest code?
> > 2. succeeded running the tutorial and get a decent result?
> > 3. Can't see any val loss on the graph. Is it because your weights is overfitting so that val loss is too high to see?
> 
> thanks, there are something wrong with my datasets.
> I have already fixed that

How did you solve that bro?
![image](https://user-images.githubusercontent.com/34357479/80911366-5deab700-8d68-11ea-90f8-5f58e233d423.png)

**Comment by peternara** [comment]
I have this problem
solve it ??

```
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.002
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.003
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.004
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.005
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.004
```

**Comment by llllliu1** [comment]
hi，i have met the same problem，how did u solve that？
